webpack.config.js

```diff
@@ -4,13 +4,24 @@ const MiniCssExtractPlugin = require("mini-css-extract-plugin");
 const CssMinimizerPlugin = require("css-minimizer-webpack-plugin");
 const TerserPlugin = require("terser-webpack-plugin");
 const libraryName = "Crumbs";
+const CopyPlugin = require("copy-webpack-plugin");
 
 module.exports = {
-  entry: ["./src/main.js"],
+  entry: ["./src/main.ts"],
   mode: "none",
-  plugins: [new MiniCssExtractPlugin({ filename: "[name].css" })],
+  plugins: [
+    new MiniCssExtractPlugin({ filename: "[name].css" }),
+    new CopyPlugin({
```

Comment:
If the `"definitions": true` thing works out then I don't think this will be needed any more :)